### DISCUSSION TITLE
New in version 6.0

### DISCUSSION THREAD

**Discussion by ansibleguy76** [ansibleguy76/ansibleforms#239] [Announcements]
Currently very busy with v6
- Vue2 -> Vue3
- Bulma -> Bootstrap 5.3
- Backend remains the same

New:
- 3 themes / light, dark, color with adjustable CSS file for the main colors

Breaking: Although most things will work as before, some bulma classes will need to be converted (like column size, tile classes, etc...)  but the core functionality will be the same.
The query type (now enum)  is already deprecated and will disappear.

Will need beta testers

**Comment by ansibleguy76** [comment]
Also added are 5 options on roles:

- showDesigner
- showSettings
- showLogs
- showDebugButtons
- showExtravars

this means you will be able to create your own designer role for example.  Or even can leave the admin role empty, although the admin role remains as a fallback.  If multiple roles apply, the most restrictive will win, which means if you set showLogs: false on the **public role**, no-one will be able to see the logs, not even the admin.

**Reply by ansibleguy76** [reply]
this is now also in 5.0.8 (beta)

**Comment by tovoro** [comment]
Im happy to beta test!

**Comment by revolonl** [comment]
Let test it ;-)
is there some things to do before install on an existing 5.xx version ?

**Reply by ansibleguy76** [reply]
The upgrades should be seamless.  However, since I'll be using bootstrap instead of bulma, the look and feel will be slightly different and if you have been using styleclasses (tile colors, group colors, columnwidths ...), these might need to be updated.

**Comment by tovoro** [comment]
Any news here? 
Happy new year :)

**Reply by ansibleguy76** [reply]
happy new year.
I'm actually pretty close, but I need to track 2 versions now. I have only the joblog to finish and then test test test obviously.
however a lot of goodies go in 5.0.8 beta as well.  git branch, extended role options (showDesigner, showLogs, showSettings, ...).  Also a bunch of fixes

**Reply by ansibleguy76** [reply]
Version 6 beta will be coming soon.  I'm very close to finishing the rewrite to vue3 and bootstrap 5.3

**Comment by revolonl** [comment]
Hi all,
this time i have a some time to test the BetaV6,
One little question, can i use my existing docker-compose file or not ?

**Reply by ansibleguy76** [reply]
Yes, if you haven't mounted the custom.js file.
I would advise going to mysql 8.4 or so.

If you want to mount logo & background => already in Q&A :)

**Reply by ansibleguy76** [reply]
ps : wait for another hour... just publishing new version :)

**Reply by revolonl** [reply]
ok thanks !

**Reply by ansibleguy76** [reply]
new version published... it now embeds multi awx configs and also backup restore functionality.  I'm also moving slowly to api v2, with a more standard approach and proper 404, 500 errors.  Backup works fine at my end, but another test has ssl issues... that's still to be sorted
